packages/spor-react/src/datepicker/DateTimeSegment.tsx

```diff
@@ -24,6 +24,8 @@ export const DateTimeSegment = forwardRef<HTMLDivElement, DateTimeSegmentProps>(
       state,
       ref as RefObject<HTMLDivElement>
     );
+    
+    const segmented = segment.type === "year" ? segment.text.slice(-2) : segment.text;
```

Comment:
Should we only have two digits for the year? Does this work with arrow up and down?